### PR TITLE
Create latn_qwerty_tly.xml

### PR DESCRIPTION
**Pull request by vedamanavi**

### CODE REVIEW

File: srcs/layouts/latn_qwerty_tly.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <key key0="u" key2="8" key3="û"/>
+    <key key0="i" key2="9" key3="*"/>
+    <key key0="o" key1="ö" key2="0" key3="("/>
+    <key key0="p" key3=")"/>
+    <key key0="q" key3="&amp;" key4="ğ"/>
+  </row>
+  <row>
+    <key shift="0.5" key0="á" key2="`" key4="a"/>
+    <key key0="s" key2="~"/>
+    <key key0="d" key1="{" key4="loc "/>
```
Comment (Julow) [inline srcs/layouts/latn_qwerty_tly.xml:20]: This `key4` doesn't look useful.

File: srcs/layouts/latn_qwerty_tly.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <key key0="t" key2="6" key3="%"/>
+    <key key0="y" key2="7" key3="^"/>
+    <key key0="u" key2="8" key3="û"/>
+    <key key0="i" key2="9" key3="*"/>
+    <key key0="o" key1="ö" key2="0" key3="("/>
+    <key key0="p" key3=")"/>
+    <key key0="q" key3="&amp;" key4="ğ"/>
+  </row>
+  <row>
+    <key shift="0.5" key0="á" key2="`" key4="a"/>
```
Comment (Julow) [inline srcs/layouts/latn_qwerty_tly.xml:18]: You should drop this `shift="0.5"` attribute:

shot

(some symbols are added by my own configuration)

**Review by Julow — COMMENTED** [review]
Some keys are present twice on the layout: a, c, g, j, q.

**Comment by Julow** [comment]
Ping. I plan to release soon and this could be part of it if it's ready.

**Comment by vedamanavi** [comment]
> Ping. I plan to release soon and this could be part of it if it's ready.

Hi. Thanks for your support. I kindly request that you keep duplicate letters if possible, because It has not been long since Talysh-speaking people began writing in their language. In the southeast corner of each key, I have placed the corresponding Azerbaijani letter for each Talysh letter. Including duplicate letters will assist Talysh speakers in writing in their language, as most Talysh people are educated in the Azerbaijani language. However, if you believe that duplicate letters should not be included on the keyboard, I will respectfully abide by your decision.

**Comment by Julow** [comment]
Hi! If the duplicated keys are intended, then it's completely fine!

**Comment by vedamanavi** [comment]
> Ping. I plan to release soon and this could be part of it if it's ready.

Thanks for your help and kindness. I have now applied the suggestion in the conversation and the layout is ready now. I would be grateful if you add in the next release.

**Comment by Julow** [comment]
The layout looks nice, thanks :)
I've updated the generated files and pushed to your branch.

**Pull request merged**